### ISSUE TITLE
Could not resolve "alpinejs"

### ISSUE THREAD

**Issue by pehrlich2** [nusserstudios/tailbliss#94]
Hello!

Trying to get started with a new Hugo site here + tailbliss.  When I run `hugo server` after setting up the submodule for the first time, I get the error: 

```
 % hugo server
WARN 2023/01/20 10:22:14 Module "tailbliss" is not compatible with this Hugo version; run "hugo mod graph" for more information.
Start building sites …
hugo v0.110.0+extended darwin/arm64 BuildDate=unknown
Error: Error building site: JSBUILD: failed to transform "js/alpine.js" (text/javascript): "/Users/peterehrlich/projects/zpc-site/themes/tailbliss/assets/js/alpine.js:1:19": Could not resolve "alpinejs"
Built in 629 ms
```

I also hopped in the `themes/tailbliss` and ran the `npm install` / `npm run build`, but got the same errors.  

I'm probably missing something a bit obvious here?

Thanks!

edit: `npm install alpinejs` seemed to fix this.  Maybe it should be an explicit dependency somehow?

**Comment by nusserstudios** [comment]
I did a merge recently. Let me take a look. I'll move scripts to Hugo modules, and start working on some issues.

**Comment by nusserstudios** [comment]
OK, try that. It was definitely the max version in the config.toml. I bumped the min down to 104.0, because that is the lowest it will go, from research. Try again. I use yarn for my package manager vs. npm (hence no package-lock.json file). Please pull the latest and let me know.

**Comment by pehrlich2** [comment]
Hmm.  I uninstalled `alpinejs`, and re-cloned the repo as indicated in the readme, but got the same Issue.  However re-running `npm install alpinejs` did get it working for me again

**Comment by nusserstudios** [comment]
Yeah, I not getting the same thing. I even ran it with node 16.18. What version of node are you running?

**Comment by pehrlich2** [comment]
```
% node -v
v19.4.0
```

**Comment by nusserstudios** [comment]
Thanks, but I still can't authenticate the issue. I get a Status 304 on localhost. with node v19.4.0. Does your config.toml file still have a max = 105.0?

**Comment by nusserstudios** [comment]
No worries, thank you for your help. I will setup some Go modules and import Alpine that way, since it's moving to a theme. Thanks for the help though, I really do appreciate you taking the time.

**Comment by nusserstudios** [comment]
I'll leave this open until we resolve it for certain.

**Comment by aleksasiriski** [comment]
I'm running into this same error on Cloudflare Pages:

```
GO_VERSION | 1.20.3
HUGO_VERSION | 0.110.0
NODE_VERSION | 19.4.0
PYTHON_VERSION | 3.7
```

```sh
Error: Error building site: JSBUILD: failed to transform "js/alpine.js" (text/javascript): "/opt/buildhome/repo/themes/tailbliss/assets/js/alpine.js:1:19": Could not resolve "alpinejs"
```

**Comment by nusserstudios** [comment]
Can you check and see if it loads from the CDN?

<script src="//unpkg.com/alpinejs" defer></script>

**Comment by nusserstudios** [comment]
If so, I'll remove it and use it that way moving forward.

**Comment by aleksasiriski** [comment]
I've changed the command from `hugo` to `npm install alpinejs && hugo` in Cloudflare Pages run command and it works now. Why is it needed to manually install alpinejs?

**Comment by aleksasiriski** [comment]
I can see this:
![image](https://user-images.githubusercontent.com/31509435/233470123-2f4a030e-e76b-4120-831f-8aaa1d321287.png)

**Comment by nusserstudios** [comment]
Not sure, I've just used Vercel and Netlify.

**Comment by aleksasiriski** [comment]
> I can see this: ![image](https://user-images.githubusercontent.com/31509435/233470123-2f4a030e-e76b-4120-831f-8aaa1d321287.png)

It's not using `<script src="//unpkg.com/alpinejs" defer></script>`

**Comment by nusserstudios** [comment]
I'm thinking it probably needs to be moved to some Go modules, but I'm not putting a time frame when I get to it. If you'd like to go ahead and set it up and run a pull request.

**Comment by aleksasiriski** [comment]
Properly fixed in #118
